classify.py

```diff
@@ -262,35 +263,47 @@ def __call__(self, lrs, y, title='', shortname=''):
         lir.ece.plot(lrs, y, path=ece_path, on_screen=not ece_path, kw_figure=kw_figure)
 
 
-def get_pairs(X, y, authors_subset, sample_size):
+def get_pairs(X, y, X_voc, conv_ids, authors_subset, sample_size):
```

Comment:
docstr?

X = conversations
y = labels
X_voc = ?
conv_ids = conversation id per instance
authors_subset = labels to include

return: conversation pairs, labels 0 or 1, vocalize score

Reply:
done